ghub.py

```diff
@@ -610,13 +610,47 @@ def create_issue(issue_text=None):
         print("Sorry, something bad happened: " + str(result))
 
 
+def close_issue(number, issue_text=None):
+    """
+    Closes an opened issue.
+
+    PATCH /repos/:owner/:repo/issues/:number
+
+    """
+    (upstream_user, upstream_repo) = get_user_and_repo('upstream', 'origin')
+    if issue_text:
+        data = json.dumps({'body': issue_text}).encode('utf8')
+        url = GITHUB_API_URL + '/repos/%s/%s/issues/%d/comments' % (
+            upstream_user, upstream_repo, number)
+        result = make_github_request(
+            url, data, headers={'content-type': 'application/json'})
+        if 'body' in result:
+            pass
+        else:
+            print("Something bad happened: " + str(result))
```

Comment:
Should you just abort if commenting fails?

Reply:
At first, I thought about aborting, but I figured if something went wrong with the comment part, they should still close the issue. 

Now that I think about it a little more, if the user does want to comment(for a specific reason, probably important), it would be a bad experience for the comment to fail but the issue to get closed. 

So now that I think about it, I believe that it should get aborted if the comment fails. I will make the change.

Reply:
This depends on the response from @ccstolley in this comment https://github.com/ccstolley/ghub/pull/15#discussion_r159031294